app/templates/recipe.html

```diff
@@ -1,6 +1,7 @@
 {% extends "layout.html" %}
 
 {% block recipe_input %}
+    <script src="{{url_for('static', filename='recipe.js')}}"></script>
```

Comment:
Scripts should always go at the very end of the body, otherwise there's a chance the elements the script is supposed to be modifying haven't loaded yet.

I'd recommend creating a `scripts` block in the base layout.html that templates can override to add whatever scripts they need. If all templates need this script, just put it in layout.html directly.

Reply:
Actually you might as well move the scripts in your <head> down to this new scripts block in layout.html as well. This will cause the elements on the page to render faster and then loading the JS afterwards. This will give the appearance of a quicker page load.

Reply:
Thanks, I updated.